app/Http/RuleSets/Album/SetAlbumHeaderRuleSet.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\RuleSets\Album;
+
+use App\Contracts\Http\Requests\RequestAttribute;
+use App\Contracts\Http\RuleSet;
+use App\Rules\RandomIDRule;
+
+/**
+ * Rules applied when Setting the header of an album.
+ */
+class SetAlbumHeaderRuleSet implements RuleSet
+{
+	/**
+	 * {@inheritDoc}
+	 */
+	public static function rules(): array
+	{
+		return [
+			RequestAttribute::ALBUM_ID_ATTRIBUTE => ['required', new RandomIDRule(false)],
```

Comment:
```suggestion
			RequestAttribute::ALBUM_ID_ATTRIBUTE => ['required', new RandomIDRule(true)],
```

Otherwise that would also include smart albums